registry.yaml

```
@@ -432,3 +432,22 @@ plugins:
         source: k8s_audit
       extraction:
         supported: true
+  - name: falco_journald_plugin
```

```suggestion
  - name: journald
```

https://github.com/falcosecurity/plugins?tab=readme-ov-file#registering-a-new-plugin

> The plugin name must match this [regular expression](https://en.wikipedia.org/wiki/Regular_expression): ^[a-z]+[a-z0-9-_\-]*$ (however, its **not recommended** to use _ in the name, unless you are trying to match the name of a source or for particular reasons)

Furthermore, why do we need to use `falco` prefix and `plugin` suffix? Could we avoid them?

---

Moreover, it has to match with the actual plugin name returned by the plugin API, which is `journald`, isn't it? :thinking: 
https://github.com/gnosek/falco-journald-plugin/blob/16137d0d9f947248dbc61acf9e4c28e0b7f494e4/src/lib.rs#L34

---

Yeah. Not sure why I thought I left the crate name as the plugin name. Will update.